crates/ra_ide_api/src/display/description.rs

```diff
@@ -0,0 +1,92 @@
+use ra_syntax::{
+    ast::{self, NameOwner, VisibilityOwner, TypeAscriptionOwner, AstNode},
+};
+
+pub(crate) trait Description {
```

Comment:
Let's rename the trait and the method to `short_label`, that seems to better align with semantics.

Reply:
Also, I like the trait based design!

Reply:
Fixed